config/webhook/kustomizeconfig.yaml

```diff
@@ -4,18 +4,18 @@ nameReference:
 - kind: Service
   version: v1
   fieldSpecs:
-  - kind: MutatingWebhookConfiguration
-    group: admissionregistration.k8s.io
-    path: webhooks/clientConfig/service/name
+  # - kind: MutatingWebhookConfiguration
+  #   group: admissionregistration.k8s.io
+  #   path: webhooks/clientConfig/service/name
   - kind: ValidatingWebhookConfiguration
     group: admissionregistration.k8s.io
     path: webhooks/clientConfig/service/name
 
 namespace:
-- kind: MutatingWebhookConfiguration
-  group: admissionregistration.k8s.io
-  path: webhooks/clientConfig/service/namespace
-  create: true
+# - kind: MutatingWebhookConfiguration
+#   group: admissionregistration.k8s.io
+#   path: webhooks/clientConfig/service/namespace
+#   create: true
```

Comment:
If not used - remove?

Reply:
i will clean up these in the next commit :D